src/utils/component-classes/index.js

```diff
@@ -24,7 +24,8 @@ export const ribbon = {
     error: 'border-red-100 bg-red-100',
     disabled: 'border-bluegray-300 bg-bluegray-300',
     sponsored: 'border-aqua-200 bg-aqua-200',
-    neutral: 'border-bluegray-300 bg-white',
+    neutral: 'border-gray-100 bg-gray-100',
```

Comment:
Why do we need both background and border of the same colour. Wouldn't it be enough to set just background color?

Reply:
Because we don't want the size of the ribbon to change between the bordered and unbordered versions - so we always have a 1px border on this component.

Reply:
Sounds like `box-sizing: border-box` on `ribbon.base` class should solve the same problem, i.e. ensuring the size of the ribbon is same regardless of border width. Any reason why you didn't choose that?

Reply:
Everything in Fabric has `box-sizing: border-box` by default.

Not sure what you're trying to solve for? The bordered version and the unbordered version would have a different size, even with that rule set.

Reply:
<img width="1401" alt="Screen Shot 2022-09-15 at 13 00 11" src="https://user-images.githubusercontent.com/736895/190387449-aad6b974-177a-4ec0-94e8-cd08d421dbfb.png">

Here's 3 boxes - the last one doesn't have a border, but does have the border-box rule set.

Reply:
Oh, my bad 🙈